addons/sale/models/sale_order.py

```diff
@@ -748,9 +748,17 @@ def _find_mail_template(self):
             return self._get_confirmation_template()
 
     def _get_confirmation_template(self):
+        """ Get the mail template sent on SO confirmation (or for confirmed SO's)
+
+        :return: `mail.template` record or None if default template wasn't found
+        """
         return self.env.ref('sale.mail_template_sale_confirmation', raise_if_not_found=False)
 
     def action_quotation_sent(self):
+        """ Mark the given draft quotations as sent.
```

Comment:
```suggestion
        """ Mark the given draft quotation(s) as sent.
```
You sometimes put parenthesis, sometimes you don't. Maybe uniformize all of it? :man_shrugging: